bdsx/bds/packets.ts

```diff
@@ -386,7 +386,18 @@ export class GuiDataPickItemPacket extends Packet {
 
 @nativeClass(null)
 export class AdventureSettingsPacket extends Packet {
-    // unknown
+    @nativeField(uint32_t)
+    flag1: uint32_t;
+    @nativeField(uint32_t)
+    commandPermission: uint32_t;
+    @nativeField(uint32_t, 0x38)
+    flag2: uint32_t;
+    @nativeField(uint32_t)
+    playerPermission: uint32_t;
+    @nativeField(ActorUniqueID)
+    actorId: ActorUniqueID;
```

Comment:
Specifically in the adventure settings packet Mojang changed the uint32 unique identifier into a signed little endian long. Is this accounted for somewhere?